spec/tapioca/compilers/dsl/protobuf_spec.rb

```diff
@@ -32,10 +32,7 @@ class Tapioca::Compilers::Dsl::ProtobufSpec < DslSpec
         Cart = Google::Protobuf::DescriptorPool.generated_pool.lookup("MyCart").msgclass
       RUBY
 
-      assert_equal(
-        ["Cart", "Google::Protobuf::Map", "Google::Protobuf::RepeatedField"],
```

Comment:
I don't love losing the check for these but 1. I didn't want to add a minimum version dependency for `google-protobuf` for users and 2. Didn't want to list all excludable constants apart from `Map` and `RepeatedField`. However, that's also a possibility.

Reply:
You can always add:
```ruby
assert_includes(gathered_constants, "Google::Protobuf::Map")
assert_includes(gathered_constants, "Google::Protobuf::RepeatedField")
``` 
as additional checks.